### DISCUSSION TITLE
ch-image build inside a docker container

### DISCUSSION THREAD

**Discussion by MFTabriz** [hpc/charliecloud#1393] [General]
I get 
`ch-run[1102]: error: can't init user+mount namespaces: Operation not permitted (ch_core.c:401 1)`
while trying to run `ch-image build --force` (v0.27) inside an unprivileged docker container. Is there any workaround for this?

**Comment by reidpr** [comment]
My *guess* is that there is a workaround, because user namespaces can be nested, and I'd guess it has something to do with Docker's configuration.
